src/screens/LoginPage/LoginPage.tsx

```diff
@@ -85,8 +95,8 @@ function LoginPage(): JSX.Element {
     }
   };
 
-  const signup_link = async (e: ChangeEvent<HTMLFormElement>) => {
-    e.preventDefault();
+  const signup_link = async () => {
+    // e.preventDefault();
```

Comment:
Remove the commented code if no longer required

Reply:
Okay, I 'll do that